docs/proposals/rollout/rollout.md

```diff
@@ -387,68 +397,19 @@ type TrafficAnalysis struct {
 
 type Metric struct {
     // Name of the metric.
-    // User also can use name point to custom metric checks.
+    // The metrics now used in kurator are `request-success-rate` and `request-duration`
+    // `request-success-rate` calculates the ratio of successful request to total request for a 
+    // checkIntervalSeconds time. It returns a value from 0 to 100.
+    // `request-duration` returns the elapsed time for the longest request within a CheckIntervalSeconds.
```

Comment:
`the longest request`,  i am sure this is not right

`CheckIntervalSeconds` is not consitent with below `IntervalSeconds`